block-explorer/v3.yml

```diff
@@ -0,0 +1,589 @@
+          description: Hash of global snapshot to return
+          required: true
+          schema:
+            type: string
+      responses:
+        200:
+          description: Global snapshot for given hash
+          content:
+            'application/json':
+              schema:
+                $ref: '#/components/schemas/Snapshot'
+
+        404:
+          description: Not found
+          content:
+            'application/json':
+              schema:
+                $ref: '#/components/schemas/NotFound'
+  /global-snapshot/{ordinal}:
+    get:
+      tags:
+        - Global snapshot
+      summary: Get global snapshot by ordinal
+      parameters:
+        - name: ordinal
+          in: path
+          description: Ordinal of global snapshot to return
+          required: true
+          schema:
+            type: integer
+            format: int64
+      responses:
+        200:
+          description: Global snapshot for given ordinal
+          content:
+            'application/json':
+              schema:
+                $ref: '#/components/schemas/Snapshot'
+        404:
+          description: Not found
+          content:
+            'application/json':
+              schema:
+                $ref: '#/components/schemas/NotFound'
+  /block/{hash}:
+    get:
+      tags:
+        - Block
+      summary: Get block by hash
+      parameters:
+        - name: hash
+          in: path
+          description: Hash of block to return
+          required: true
+          schema:
+            type: string
+      responses:
+        200:
+          description: Block for given hash
+          content:
+            'application/json':
+              schema:
+                $ref: '#/components/schemas/Block'
+
+        404:
+          description: Not found
+          content:
+            'application/json':
+              schema:
+                $ref: '#/components/schemas/NotFound'
+  /transaction/{hash}:
+    get:
+      tags:
+        - Transaction
+      summary: Get transaction by hash
+      parameters:
+        - name: hash
+          in: path
+          description: Hash of transaction to return
+          required: true
+          schema:
+            type: string
+      responses:
+        200:
+          description: Transaction for given hash
+          content:
+            'application/json':
+              schema:
+                $ref: '#/components/schemas/Transaction'
+        404:
+          description: Not found
+          content:
+            'application/json':
+              schema:
+                $ref: '#/components/schemas/NotFound'
+  /global-snapshot/latest/transaction:
+    get:
+      tags:
+        - Transaction
+      summary: Get transactions from the latest snapshot
+      parameters:
+        - name: limit
+          in: query
+          description: Limit (size) of the response items
+          required: false
+          schema:
+            type: integer
+        - name: search_after
```

Comment:
Won't all transactions have the same timestamp in the `GlobalSnapshot` so searching by it will not give us anything? All transactions will either make it or not make it through the filter, right?

Reply:
Transactions from the single Global snapshot will have the same timestamp.

Reply:
So I would remove filtering by timestamp for this endpoint as it gives nothing.

Reply:
hmm, I will think about that